apps/google-analytics-4/frontend/src/components/sidebar/SidebarAnalytics.tsx

```diff
@@ -0,0 +1,46 @@
+import { useAutoResizer, useFieldValue } from '@contentful/react-apps-toolkit';
+import ChartFooter from 'components/sidebar/ChartFooter';
+import ChartHeader from 'components/sidebar/ChartHeader';
+import { useEffect, useState } from 'react';
+import { RunReportResponse } from 'types';
+import { config } from '../../config';
+
+export default function SidebarAnalytics() {
+  const [runReportResponse, setRunReportResponse] = useState<RunReportResponse | undefined>();
+  const [slugValue] = useFieldValue<string | undefined>('slug');
```

Comment:
I'm pretty sure you can get rid of the `undefined` typing because your slugValue will never be undefined since you set the value here and never update it anywhere else